### PR TITLE
House keeping 🧹 

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Some house keeping for this project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,13 +83,14 @@
     }
   },
   "scripts": {
+    "all": "run-s test",
```
Comment (MaxTru) [inline package.json:86]: Why would `all` not also `build`/ `compile`?

**Review by MaxTru — COMMENTED** [review]
LGTM! Just one question

File: package.json (hunk 1 above)
Comment (pinussilvestrus) [reply package.json:86]: It's doing it in [`pretest`](https://github.com/bpmn-io/vs-code-bpmn-io/blob/master/package.json#L92). I think this comes from the extension boilerplate we were using once we created it. Probably the VSCode environment uses this `pretest` task.

Comment (MaxTru) [reply package.json:86]: ah - I see

**Review by MaxTru — APPROVED** [review]

**Pull request merged**